tests/dcosjob.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
What's the underlying function that the dcos cli is using to check that a job is finished?  If that exists in the `dcos` package, let's use that.  Otherwise, CLI is fine.